manifests/role/ansible_master.pp

```diff
@@ -36,20 +36,22 @@
   $host_groups,
   $playbooks_source = 'https://github.com/openshift/openshift-ansible.git',
   $playbooks_version = 'master',
+  $ansible_inventory = '/etc/ansible/hosts',
+  $packages = [
+    'ansible',
+    'git',
+    'jq',
+    'pyOpenSSL',
+    'wget',
+  ],
 ) {
   validate_hash($host_groups)
```

Comment:
Missing `validate_absolute_path($ansible_inventory)` (feel free to add validations for the other parameters in a separate commit).